Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -94,14 +103,15 @@ SteamScript.prototype.go = function () {
         .to(tempFile)
         .get();
 
-    setupWizard.wait(tr("Please follow the steps of the Steam setup.\n\nUncheck \"Run Steam\" or close Steam completely after the setup so that the installation of \"{0}\" can continue.", this._name));
-
     var wine = new Wine()
         .wizard(setupWizard)
         .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
         .luna();
+    
+    setupWizard.wait(tr("Please follow the steps of the Steam setup. After updating and loging in, shut down completly Steam client so the installation of \"{0}\" can continue.", this._name));
```

Comment:
I would prefer:
"Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of ..."